EfEnumToLookup/LookupGenerator/MetadataHandler.cs

```diff
@@ -1,267 +1,318 @@
 ﻿namespace EfEnumToLookup.LookupGenerator
-{
```

Comment:
large amount of diff-noise here probably due to indent / line-ending differences, fix that up and I'll have another look